packages/pharos/src/components/table/pharos-table.ts

```diff
@@ -105,6 +118,52 @@ export class PharosTable extends ScopedRegistryMixin(PharosElement) {
     this.totalResults = !this.totalResults ? this.rowData.length : this.totalResults;
   }
 
+  private _toggleActiveStickyHeader = (active: boolean) => {
+    const ACTIVE_HEADER_CLASS = 'table-sticky-header--is-active';
+    const ACTIVE_HEADER_CELL_CLASS = 'table-sticky-header__cell--is-active';
```

Comment:
I am curious if you need an active cell class or if your sass can just style the children cells of the sticky header active class? Something like...
```sass
.table-sticky-header--is-active {
  ... styles
  th {
    ... styles
  }
}
```
... hard to know if this is possible as I cannot see the HTML in this PR. It is also more of nit, it's not like querying all the th elements is a very big deal.

Reply:
👍 That was done in an [effort to be more BEM-like](https://github.com/ithaka/pharos/pull/813#discussion_r1742724972)